packages/fuels-types/src/param_types.rs

```diff
@@ -310,7 +319,8 @@ fn try_vector(the_type: &Type) -> Result<Option<ParamType>, Error> {
 }
 
 fn try_enum(the_type: &Type) -> Result<Option<ParamType>, Error> {
-    let result = if has_enum_format(&the_type.type_field) {
```

Comment:
Why weren't all those methods kept in the `fuel-abi-types` repo? Those utils are useful and avoid having to define helpers in all downstream crates.